content/javascript/concepts/setTimeout/setTimeout.md

```diff
@@ -0,0 +1,91 @@
+---
+Title: 'setTimeout'
```

Comment:
```suggestion
Title: 'setTimeout()'
```

Reply:
A little tricky, we'll have to wait to merge this PR until the window concept in issue [#2007](https://github.com/Codecademy/docs/issues/2007) is merged.

But we'll handle this when the time comes or when the mentioned issue's PR is merged you can feel free to do this yourself; this folder/file will go into a folder named `window/terms/`.